### ISSUE TITLE
[#SomeChanges Suggestion] Cross-realm queuing

### ISSUE THREAD

**Issue by perpabc** [JamminL/wotlk-classic-bugs#179]
As the restricting of character creation & transfers to larger servers become a staple of Classic WoW versions, people are struggling to team up with their friends & guildies as they return to the game to find they're on the 'wrong' server. There was recently a post where you outlined the importance of the social aspect of the game, and in tune with this I strongly suggest allowing people to queue Arena together even if they're on different servers.

**Comment by Evolvee** [comment]
I just want to "refresh" this topic and add the following, regarding the recently made posts of Aggrend about the server queues and the over-populated so-called "megaservers", which is a hot topic atm and even according to Blizzard, its the #1 issue that is _"being worked on."_

First of all,

https://us.forums.blizzard.com/en/wow/t/classic-the-unacceptable-state-of-classic-servers/1323722/7
https://us.forums.blizzard.com/en/wow/t/classic-the-unacceptable-state-of-classic-servers/1323722/10

both of these posts only talk only about issues of players not being able to form groups for dungeons, raids and the problems with guild social circles. They also go on and on about how bigger the currently presented "dead servers" are compared to 2008 servers. While its obvious that the player type described there (PVErs, casuals, dadgamers & whales) take the absolute majority of the entire playerbase, these posts, in my opinion, **COMPLETELY omit** one "key" fraction of the overall community: **The Arena Playerbase**.

Second of all, let me simplify the issue arena players are dealing with here and why its impossible to ask any arena players to move to other servers and leave their current "megaserver" behind:

# NOT A SINGLE SEMI-SERIOUS ARENA PLAYER WILL EVER TRANSFER FROM GEHENNAS (for HORDE) OR FIREMAW (for ALLIANCE) AS LONG AS ITS NOT POSSIBLE TO QUEUE ARENAS TOGETHER WITH PEOPLE OUTSIDE OF THESE REALMS.

_^^ That applies as long as such player doesn’t already have a hard-set arena partner(s) to play with for the entire length of the expansion._

**Why?**

# 1. The inability to queue with good arena partners.
If you are on "above average" level of arena gameplay and enjoy competitive environment and want to push forward, you simply don’t want to waste your time with people who are way below your level. You want to queue with ppl who are on par or even better than you are. For people who are competing in the high-end arena scene, no such ppl exists on other servers than these "megaserver" ones.

# 2. The scale of potential arena partners to play with.
Obviously, if you are on a mega-server that is not described above (mainly Gehennas, Firemaw), you are severely limiting yourself to who you can play with. This may not be as big issue in 2s bracket, but its impact will significantly increase in 3s and 5s brackets. With the removal of teams in Wrath of the Lich King Classic™®, its obvious that Blizzard wants to allow people to play with more different players than the number of players that can fit in a set team bracket (4 for 2s, 6 for 3s, 10 for 5s). Such effort is however completely thwarted in case there are no players to actually play with. The removal of teams "benefits" only players who play on these so called "megaservers" way more than ppl who take the plunge and transfer off elsewhere.

# 4. The risk of a realm dying & its consequences.
I can easily describe this experience first-hand. In season 1 of Classic TBC, me and a bunch of other players started to play on the realm "Earthshaker (EU, horde)", simply because we were scared that Gehennas (EU, horde) would be a queue-simulator and we wouldn’t be able to play the game from the beginning and due to that, fall behind. This turned out not to be the case, however we were able to play with each other so we didn’t miss anything, YET. The obvious issue started to arise around mid-season already. The server started to lose players either due to transfers or simply due to the initial hype going away. At the end of season 1, it was almost impossible to even find a raid group for the T4 content - now imagine how hard it would be for any arena players to find a new team partner there if they didn’t already have a long-time arena partner to play with. This scenario happened EVERYWHERE except on the megaservers - even a server that was literally nothing but full of arena players all in one guild - Stonespine (Eu,Horde) - died off _(this could partially be due to the lack of clients to boost there and the "fail" of their PVE division to secure any world-firsts, but whatever)_.

That is only the one side of the problem though, another .. and probably even bigger problem .. is the fact that unless you paid a substantial amount of real-money to transfer off such realm, the only other possibility for an arena player playing on Earthshaker EU, horde, at the end of first TBCC season was to make a brand-new character elsewhere (mainly Gehennas) - which completely deletes all achievements, progress and character bonding that happened during the entire season. I don’t want to waste too much time here but its fair to say that we all know how disgusting the paid transfer were during Classic TBC, I would go even as far as saying the paid transfers bordering with actual **predatory practices**, especially when it comes to players with multiple characters who were literally forced to pay such a horrendous amount of money for such a simple and fully automated service for EACH of their characters individually. Either the goal there was to milk the playerbase to make Bobby&Shareholders PeepoHappy _(cant rly put any blame there, its, unfortunately, a publicly traded company after all, if it wasnt trying to make money it could get sued)_, or it was simply an incompetence on the company´s part to realize and distinguish a pressing issue its players are dealing with - no matter what, I think the culprit in the eyes of the entire player community is pretty clear here. The "free transfers" and the "server consolidation" both came too late to solve anything - both came months after the issue was already "solved" by the players themselves (at the expense of their and their moms’ wallets).

After this intimidating experience, you simply cannot expect arena players to take the risk and intentionally put themselves into the position described above once again. Unless you give arena players a clear assurance that no similar situation will ever happen if they decide to transfer, players will (understandably) not do it.

**TLDR of Paragraph 4:** [https://youtu.be/rQ6N-sb7SVQ](https://youtu.be/rQ6N-sb7SVQ)

------------------------------------------

**Pros of Cross-realm arena queue:**
- Arena-oriented players will be able to transfer off "mega-servers", its a win-win for everybody.
- More players participating in the arena, creating a bigger scene.
- Its harder to snipe certain players when they are queuing with people outside of their own realm - #271 
- Players will have a soft-assurance that no matter what happens to their server, they can always participate in PVP (both BG-wise & arena-wise)

**Cons of Cross-realm arena queue:**
- RMT-players will get more clients to boost and wintrade, essentially screwing the end of season ladder even more than ever before _(tbh its already so bad, I don’t think this change could make it any significantly worse, especially if Blizzard keeps allowing such behavior to continue due to badly set rules and the inability of GM team to monitor anything but AV afkers via an automated metrics system)_
- Some RP-PVP "players" feelings could be hurt, for no reason...

**Comment by ManneN1** [comment]
I guess this can be closed now, considering it's being implemented tomorrow. @JamminL
